src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/X509Pal.OpenSsl.cs

```diff
@@ -5,9 +5,6 @@ namespace System.Security.Cryptography.X509Certificates
 {
     internal sealed partial class X509Pal
     {
-        private static partial IX509Pal BuildSingleton()
-        {
-            return new OpenSslX509Encoder();
-        }
+        private static partial IX509Pal BuildSingleton() => new OpenSslX509Encoder();
```

Comment:
NIT
```suggestion
        private static partial OpenSslX509Encoder BuildSingleton() => new OpenSslX509Encoder();
```

Reply:
The return type here can't be changed as it's a partial method and the signature of all of the partials need to match, but the partial declaration is in the cross-platform shared file.